src/ER/Beds.js

```diff
@@ -1,54 +1,117 @@
 import React, { useState } from 'react';
+import Counter from '../Components/counter';
+import Banner from '../Components/banner';
+import ButtonRight from '../Components/ButtonRight';
+import DiversionPopup from '../Components/DiversionPopup';
+import DiversionPopupConfirmation from '../Components/DiversionPopupConfirmation';
+import './hospitalinfo.css';
```

Comment:
To make it easier to see the connection between the component and its styles, let's name the css file as Beds.scss instead of hospital info. Then it will live right next to Beds.js in the file browser.

Note that using the scss extension instead of css will let you use SASS mixins and enhanced syntax.